packages/e2e-tests/fixtures/blocks/core__image__center-caption.json

```diff
@@ -4,10 +4,10 @@
         "name": "core/image",
         "isValid": true,
         "attributes": {
-            "url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==",
+            "align": "center",
+	        "url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==",
```

Comment:
weird spacing

Reply:
Good catch, there was a conflict to resolve when rebasing so I did something wrong. I will fix it.

Reply:
Fixed